source/overview/compliance.rst

```diff
@@ -172,21 +165,24 @@ advice or to be used as a substitute for specific legal advice. You should not a
 information on this site without obtaining professional advice regarding particular facts and circumstances.
 
 FAQ
------------------------------------------------
+----
 
 To be compliant with GDPR, do I need to remove message contents of email notifications?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Based on our interpretation of GDPR, it is not required to hide message contents in email notifications to remain compliant for the following reasons:
 
 1. Every user has the ability to disable email notifications in their Account Settings. Therefore, every user has the ultimate control over whether or not they want information sent via email. This option aligns with most other products, but we will follow updates on interpretations of GDPR closely to see if we need to make changes in this area.
-
 2. Mattermost offers :ref:`TLS encryption <email-tls>` to protect communication between the Mattermost server and the SMTP email server.
-
 3. For those who are uncertain if the first two points cover GDPR compliance, we offer the ability to :ref:`disable notifications completely <email-notification-config>` on your Mattermost server. To use Mattermost in production with no email notifications, you also need to :ref:`disable a "preview mode" notice banner <email-preview-mode-banner-config>`.
 
+What does Mattermost do with my email address/contact details?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you contact Mattermost using a **Contact Us** button, website form, or other method and supply your email address and contact details, that information is sent to us directly and used to contact you as requested.
```

Comment:
For the "Contact Support" button for admin advisor notifications, the admin doesn't supply the email address, it's automatically sent to us.